source/index.html.erb

```diff
@@ -19,25 +19,26 @@ title: RailsBridge Boston - Ruby and Rails workshop
   </div>
 </section>
 
-<div class="actionable-sections">
-  <section class="next-workshop">
-    <h2 class="section-title">Next workshop</h2>
-    <h2 class="workshop-title"><%= next_event.title %></h2>
-    <p class="workshop-date"><%= next_event.dates %></p>
-    <div class="workshop-description"><%= next_event.description %></div>
+<section class="upcoming">
+  <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2948.1159095133034!2d-71.08135500000003!3d42.36136900000001!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89e370a423d61825%3A0x58516248462c99eb!2sMicrosoft+New+England+Research+and+Development+Center!5e0!3m2!1sen!2sus!4v1425584974378" width="300" height="225" frameborder="0" style="border:0"></iframe>
```

Comment:
How about moving this URL up somewhere else instead of in the middle of the html? It would be pretty hard to find it.